src/group.coffee

```diff
@@ -239,3 +240,22 @@ module.exports = (robot) ->
       res.send "#{name} is in #{group.membership(name).join ", "}."
     else
       res.send "#{name} is not in any groups!"
+
```

Comment:
Moving the `group add` command down here is making a big diff for no reason -- could you move this edited block back up to where it was before (between `group rename` and `group remove`).